client/components/club-details/ClubInfo.tsx

```diff
@@ -0,0 +1,99 @@
+import { useRouter } from 'next/router'
+import styled from 'styled-components'
+import { Club } from '../../context'
+import { fontWeight, PageTitle } from '../../styles'
+import { Category } from '../Category'
+
+const BackArrow = styled.img`
+  margin-top: 64px;
+  cursor: pointer;
+`
+
+const Logo = styled.img`
+  border-radius: 100%;
```

Comment:
Ohh I see. I got confused because Figma's design is the Tech+ logo which I thought was rounded. @CarolinaLi should the logo on club pages be squares like the club list or circles?

![image](https://user-images.githubusercontent.com/14998054/117006542-93491e00-acb6-11eb-9a97-0aae2cfe2d3a.png)

Reply:
It should be squares to maximize space! If the logo itself is rounded, it could fit into the square and appear as a circle. Adding rounded corners to the square logo would be nice too, maybe 4px? 

![image](https://user-images.githubusercontent.com/33561866/117053181-a542b500-ace6-11eb-8c52-800b28f14ace.png)

Reply:
Done!